spec/temporal.html

```diff
@@ -89,7 +89,7 @@ <h1>Temporal.now.instant ( )</h1>
         The following steps are taken:
       </p>
       <emu-alg>
-        1. Return ? SystemInstant().
+        1. Return SystemInstant().
```

Comment:
I have the same comment here as Jordan wrote in the other pull request.
```suggestion
        1. Return ! SystemInstant().
```